src/main/resources/types.graphqls

```diff
@@ -38,6 +38,7 @@ input ActiveRecordingFilterInput {
 input ArchivedRecordingFilterInput {
     name: String
     labels: [String]
+    sourceTarget: String
```

Comment:
@andrewazores I think I'm good to go on that front, unless I need to add something else here?

Reply:
Oh, my bad. I could've sworn this change didn't show up in my diff viewer when I first looked at this.